astroplan/observer.py

```diff
@@ -81,15 +89,15 @@ class Observer(object):
 
     Examples
     --------
-    We can create an observer at Subaru Observatory in Hawaii two ways. First,
+    We can create an observer at Subaru Observatory in Hawaii three ways. First,
     locations for some observatories are stored in astroplan, and these can be
```

Comment:
this is not true since #168, @bmorris3 you may want to have a look and rewtrite this